move-vm-backend/tests/move_vm.rs

```diff
@@ -62,6 +64,18 @@ fn estimate_gas_for_published_bytecode(bytecode: &[u8]) -> u64 {
     num_integer::div_ceil(raw_gas_cost, 1000)
 }
 
+fn store_preloaded_with_stdlib() -> StorageMock {
```

Comment:
After merging this PR it will preload both stdlibs.

Reply:
Hm - I'll rename it to `store_preloaded_with_genesis_config` to avoid any confusion - since we might add more stuff in the `GensisConfig`, which is not just the stdlib loading

Reply:
Done